src/Microsoft.SqlTools.ServiceLayer/DacFx/Contracts/DeploymentOptions.cs

```diff
@@ -436,9 +440,23 @@ public DeploymentOptions(DacDeployOptions options)
             // Setting Display names for all dacDeploy options
             SetDisplayNameForOption();
 
+            // Preparing Include Object types options table
+            GetIncludeObjectTypesDictionary();
+
             SetOptions(options);
         }
 
+        /// <summary>
+        /// Gets include objects enum names and values as a dictionary
+        /// Ex: {key: Aggregate, value: 0}
+        /// </summary>
+        public void GetIncludeObjectTypesDictionary()
+        {
+            // Set include objects table data
+            var objectTypeEnum = typeof(ObjectType);
+            IncludeObjects = Enum.GetNames(objectTypeEnum).ToDictionary(t => t, t => (int)System.Enum.Parse(objectTypeEnum, t));
```

Comment:
using enum number is a bad idea. this means you cannot change the order of values in Enum in dacfx otherwise it can break ADs if the number is used in any extension. don't give any information you don't want any extension to use